core/zip/inc/Compression.h

```diff
@@ -39,56 +39,57 @@ namespace ROOT {
 ///   [207 - 208]
 ///  - LZ4 is recommended to be used with compression level 4 [404]
 
+class RCompressionSetting {
+   public:
+      enum class EDefaults {
```

Comment:
I think we want to keep it as a regular enum to avoid the user to have to do static_cast.

Reply:
One possibility to consider if you like the extra scope/naming
```
namespace RCompressionSetting {
   namespace Defaults {
       enum= EDefaults {
           ....
```

Reply:
Nested namespace is a bad idea if we want to make `RCompressionSetting` a class later.